src/resources/books.md

```diff
@@ -10,6 +10,24 @@ If you find another Dart book that might be helpful, please
 [let us know.](https://github.com/dart-lang/site-www/issues)
 
 
+{% for book in site.data.books-dart3 %}
```

Comment:
Instead of naming this `books-dart3` could you name it `books-null-safe` or something along those lines? In case we do have a Dart 3 one day :)

Reply:
Renamed it to `books-dart-null-safe` to stay with the `books-dart` style as well.